tsconfig.types.test.json

```diff
@@ -13,5 +13,5 @@
     "test/hotCases/**/*loader*.js",
     "declarations.test.d.ts"
   ],
-  "exclude": ["test/js/**/*"]
+  "exclude": ["/benchmark/.generated/**/*"]
```

Comment:
Please don't change it, we validate our types in tests too, just add `benchmark`